lisa/transformers/dom0_kernel_installer.py

```diff
@@ -258,3 +265,25 @@ def _update_mariner_config(
         file=mariner_config,
         sudo=True,
     )
+
+    uuid = node.execute(
+        "lsblk -o MOUNTPOINT,UUID | grep '/ ' | awk '{print $2}'",
+        sudo=True,
+        shell=True,
+        expected_exit_code=0,
+    ).stdout.strip()
+    partuuid = node.execute(
```

Comment:
Can lsblk tool do this?

Reply:
No, lsblk tool does not fetch UUID and PARTUUID.